src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -772,6 +772,14 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         if (changed) {
             logger.info(`Memberships for call in room ${this.room.roomId} have changed: emitting`);
             this.emit(MatrixRTCSessionEvent.MembershipsChanged, oldMemberships, this.memberships);
+
+            if (this.isJoined() && !this.memberships.some(this.isMyMembership)) {
+                logger.warn("Missing own membership: force re-join");
+                // TODO: Should this be awaited? And is there anything to tell the focus?
+                this.triggerCallMembershipEventUpdate();
+                // TODO: Is this needed?
+                this.emit(MatrixRTCSessionEvent.JoinStateChanged, true);
```

Comment:
As a consumer of this module I probably wouldn't expect this to be emitted. There is no change :D